hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/AbstractFileTransferHandler.java

```diff
@@ -109,17 +98,30 @@ protected void startTransferToRepositoryThread(final InputStream inputStream, fi
     }
 
     private void setFailureReason(final String failureReason) {
+        setUploadInterrupted();
```

Comment:
Remove this, because every method dealing with error is calling `setUploadInterrupted()` already